Andrews, S. 2010. “FastQC: A Quality Control Tool for High Throughput Sequence Data.” http://www.bioinformatics.babraham.ac.uk/projects/ fastqc.

Ewing, Brent, LaDeana Hillier, Michael C. Wendl, and Phil Green. 1998. “Base-Calling of Automated Sequencer Traces Using Phred. I. Accuracy Assessment.” Genome Research 8 (3) (March 1): 175–185. doi:10.1101/gr.8.3.175. http://europepmc.org/abstract/MED/9521922.

Grabherr, Manfred G, Brian J Haas, Moran Yassour, Joshua Z Levin, Dawn A Thompson, Ido Amit, Xian Adiconis, et al. 2011. “Full-Length Transcriptome Assembly from RNA-Seq Data without a Reference Genome.” Nature Biotechnology 29 (7) (May 15): 644–652. doi:10.1038/nbt.1883. http://www.nature.com/doifinder/10.1038/nbt.1883.

Krueger, F. 2013. “Trim Galore. A Wrapper Tool around Cutadapt and FastQC to Consistently Apply Quality and Adapter Trimming to FastQ Files, with Some Extra Functionality for MspI-Digested RRBS-Type (Reduced Representation Bisufite-Seq) Libraries.” http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/.


Olson, Claire; Roberts, Steven (2014): Crassostrea gigas male gonad transcriptome data comparison. figshare.
http://dx.doi.org/10.6084/m9.figshare.1004464
Retrieved 15:19, Nov 15, 2015 (GMT)
